### DISCUSSION TITLE
Indicator icons for Moderators/Admins

### DISCUSSION THREAD

**Discussion by Arcadius2006** [LBPUnion/ProjectLighthouse#695] [Ideas]
Moderators and Administrators should have icons on their profiles that show their role
![Moderator Indicator](https://user-images.githubusercontent.com/93880348/223213145-fe60050c-ac2e-4f08-a3c2-21cb54185351.png)
![Admin Indicator](https://user-images.githubusercontent.com/93880348/223213150-b760972c-543b-4aba-89c6-9918db4e77e4.png)
(concept icons)

**Comment by sudokoko** [comment]
Seems easy enough to implement with the current method of highlighting mod/admin roles, though we'd probably opt in to use the in-house icons from our theming system.

**Comment by Arcadius2006** [comment]
this is for LH in general, so we need to have non-descript icons

you can just implement an option to replace said icons on any given
instance, so beacon can use the funky lbpu icons

**Reply by sudokoko** [reply]
> this is for LH in general, so we need to have non-descript icons you can just implement an option to replace said icons on any given instance, so beacon can use the funky lbpu icons

By in house icons I mean Lighthouse's iconset (which is based upon FontAwesome). But yeah we may be able to do this as well

**Comment by Arcadius2006** [comment]
the reason i did them like this is because theres already that level
indicator issue/idea i made before which also used icons like that

wt., 7 mar 2023, 22:12 użytkownik koko ***@***.***> napisał:
